app/controllers/oidc/api_key_roles_controller.rb

```diff
@@ -134,7 +134,7 @@ def add_default_params(rubygem, statement, condition)
 
     return unless rubygem
     return unless (gh = helpers.link_to_github(rubygem)).presence
-    return unless (@api_key_role.provider = OIDC::Provider.github_actions)
+    return unless (@api_key_role.provider == OIDC::Provider.github_actions)
```

Comment:
Is the conditional with assignment instead of equality intentional or accidental? 🤔

Reply:
it intentionally was an assignment, this is where the provider would get assigned (as a default, I think)